### PR TITLE
Workshop - 0.2 to 0.3 code transform of examples

### PR DESCRIPTION
**Pull request by shaileshgrowisto**

### CODE REVIEW

**Comment by amitu** [comment]
@shaileshgrowisto, thanks for the pull request. Regarding comments, we have moved to double semicolon for comments instead of single semicolons. Can you use that and update the PR?

**Comment by shaileshgrowisto** [comment]
> @shaileshgrowisto, thanks for the pull request. Regarding comments, we have moved to double semicolon for comments instead of single semicolons. Can you use that and update the PR?

Sure @amitu , will validate and update the pull request.

**Comment by amitu** [comment]
@shaileshgrowisto is this done? Should I merge it?

**Comment by shaileshgrowisto** [comment]
> @shaileshgrowisto is this done? Should I merge it?

I have added the double semicolons, you can merge the code.

File: 01-data/01-hello-world/FPM.ftd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+;; check: https://ftd.dev/workshop/hello-world/ for instructions
+-- import: fpm
+
+-- fpm.package: hello
```
Comment (amitu) [inline 01-data/01-hello-world/FPM.ftd:4]: This file should not be checked in, we ask learner to create this file as first task.

File: 01-data/07-http-processor/index.ftd [hunk 2]
```diff
@@ -15,25 +15,25 @@ url: https://kameri-service.herokuapp.com/api/sample/
...
-padding: 10
+spacing.px: 10
+padding.px: 10
 
---- todo-view:
-t: $t
-$loop$: $todo-list as $t
+/-- todo-view:
+/t: $t
+/$loop$: $todo-list as $t
```
Comment (amitu) [inline 01-data/07-http-processor/index.ftd:23]: These should not be commented out.

File: 01-data/07-http-processor/index.ftd [hunk 3]
```diff
@@ -15,25 +15,25 @@ url: https://kameri-service.herokuapp.com/api/sample/
...
-border-width: 1
-border-radius: 5
-
-
---- ftd.text: $t.title
---- ftd.integer: $t.id
---- ftd.text: $t.status
---- ftd.text: $t.description
+/Uncomment the above code and run to see the below error
+/FPM-Error: path: /, PackageError { message: "failed to parse ParseError { message: \"hello/: not found (\\\"http\\\"), f: Variable::from_ast\", doc_id: \"hello/\", line_number: 11 }" }
```
Comment (amitu) [inline 01-data/07-http-processor/index.ftd:39]: Is this still happening?

File: 01-data/01-hello-world/FPM.ftd (hunk 1 above)
Comment (shaileshgrowisto) [reply 01-data/01-hello-world/FPM.ftd:4]: Oh, okay. I will remove this

**Comment by AbrarNitk** [comment]
@amitu @shaileshgrowisto I am merging this PR for now, please create a new PR. Please resolve the comments.

**Pull request merged**